internal/scm-clients/github/github.go

```diff
@@ -42,7 +42,7 @@ type GithubClientImpl struct {
 
 var _ GithubClient = (*GithubClientImpl)(nil) // Verify that *GithubClientImpl implements GithubClient.
 
-func InitClient(client *http.Client, token string) (GithubClient, error) {
+func InitClient(client *http.Client, token string, host string) (GithubClient, error) {
 	gc := github.NewClient(client)
```

Comment:
please check `github.NewEnterpriseClient` for supporting github self hosted servers

Reply:
Updated to include NewEnterpriseClient for non-Github.com domains